recipes/libpq/all/conanfile.py

```diff
@@ -13,7 +11,6 @@ class LibpqConan(ConanFile):
     homepage = "https://www.postgresql.org/docs/current/static/libpq.html"
     author = "Bincrafters <bincrafters@gmail.com>"
     license = "PostgreSQL"
-    exports_sources = ["CMakeLists.txt"]
```

Comment:
CMakeLists.txt itself is not removed from repo..?

Reply:
I've kept it for the compatibility, but I can remove it, if needed

Reply:
removed all the CMake leftover